test/distributed/test_distributed.py

```diff
@@ -1967,7 +1967,10 @@ def test_DistributedDataParallel_non_default_stream(self):
                 # Clear gradients manually
```

Comment:
Why is this cleared manually and not using `net.module.zero_grad()` ? That would simplify this logic.

Reply:
sounds good